.eslintignore

```diff
@@ -1,5 +1,6 @@
-.github
```

Comment:
This is unrelated to this specific PR, just something I missed in https://github.com/withastro/language-tools/pull/334